### ISSUE TITLE
Get crash on using CardField

### ISSUE THREAD

**Issue by redwind** [flutter-stripe/flutter_stripe#1442]
**Describe the bug**
Hi, 
I got crash when using CardField

Here is crash log
`
E/AndroidRuntime(24453): FATAL EXCEPTION: main
E/AndroidRuntime(24453): java.lang.NoClassDefFoundError: Failed resolution of: Lcom/stripe/android/databinding/StripeCardInputWidgetBinding;
E/AndroidRuntime(24453): 	at com.reactnativestripesdk.CardFieldView.<init>(CardFieldView.kt:36)
E/AndroidRuntime(24453): 	at com.reactnativestripesdk.CardFieldViewManager.createViewInstance(CardFieldViewManager.kt:69)
E/AndroidRuntime(24453): 	at com.flutter.stripe.StripeSdkCardPlatformView.<init>(StripeSdkCardPlatformView.kt:29)
E/AndroidRuntime(24453): 	at com.flutter.stripe.StripeSdkCardPlatformViewFactory.create(StripeSdkCardPlatformViewFactory.kt:24)
E/AndroidRuntime(24453): 	at io.flutter.plugin.platform.PlatformViewsController$1.createForPlatformViewLayer(PlatformViewsController.java:184)
E/AndroidRuntime(24453): 	at io.flutter.embedding.engine.systemchannels.PlatformViewsChannel$1.create(PlatformViewsChannel.java:109)
E/AndroidRuntime(24453): 	at io.flutter.embedding.engine.systemchannels.PlatformViewsChannel$1.onMethodCall(PlatformViewsChannel.java:60)
`

**Smartphone / tablet**
 - Device:   sdk gphone64 arm64 (mobile) • emulator-5554 • android-arm64  • Android 13 (API 33) (emulator)
 - OS: Android 13 (API 33)
 - Package version: 9.4.0
 - Flutter version 3.3.10

**Comment by remonh87** [comment]
Can you give a reproduction example and does the cardfield work when you run our example app?

**Comment by albertodenia** [comment]
@redwind I was experiencing the same crash, which was caused by incompatible framework versions. In particular we had the following version constraints in our `android/app/build.gradle` file:

```
dependencies {
    // ...
    constraints { implementation('com.stripe:stripe-android') { version { strictly '20.11.0' } } }
    constraints { implementation('com.stripe:financial-connections') { version { strictly '20.11.0' } } }
}
```

which were added some time ago as a workaround for [another issue](https://github.com/flutter-stripe/flutter_stripe/issues/1000)). They aren't needed anymore so I removed them and the crash got fixed.
